src/core/Akka.Remote/RemoteActorRef.cs

```diff
@@ -112,7 +113,16 @@ public override IActorRefProvider Provider
         /// <exception cref="System.NotImplementedException">TBD</exception>
         public override IActorRef GetChild(IEnumerable<string> name)
         {
-            throw new NotImplementedException();
+            var items = name.ToList();
+            switch (items.FirstOrDefault())
```

Comment:
Needed to resolve `ActorSelection`s on remotely deployed actors - matches the corresponding JVM code here: https://github.com/akka/akka/blob/84966a2ca54f96360a3996e1c670684e2948f870/akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala#L513